SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psd1

```diff
@@ -1,5 +1,5 @@
 @{
     ModuleVersion = '1.0'
     RootModule = 'SecretManagement.LastPass.Extension.psm1'
-    FunctionsToExport = @('Set-Secret','Get-Secret','Remove-Secret','Get-SecretInfo','Test-SecretVault')
+    FunctionsToExport = @('Set-Secret','Get-Secret','Remove-Secret','Get-SecretInfo','Test-SecretVault','Invoke-lpass')
```

Comment:
Invoke-lpass get exposed to the main module but is not exposed to the user.

Reply:
I like this for now. If folks want us to export it higher (for like adhoc commands that will be wsl/xplat agnostic) we can revisit.